_data/mentors.yml

```diff
@@ -312,7 +312,7 @@
     I am a software engineer with around 6 years of experience in software and a special focus on cloud and cybersecurity in the last few years. I design and deliver scalable Java solutions for complex business challenges in the healthcare and retail industry. I leverage my expertise in software development security, cryptography, and network security to ensure the protection and integrity of data and systems. I collaborate with cross-functional teams to develop, test, and deploy high-quality software using Agile methodologies and various frameworks and technologies.
   image: assets/images/mentors/hersi_kopani.jpg
   languages: English
-  availability: [5, 6, 7, 8, 9,10, 11]
+  availability: [5, 6, 7, 8, 9, 10, 11]
```

Comment:
@IdayatSanni remember to change [this](https://github.com/Women-Coding-Community/WomenCodingCommunity.github.io/blob/main/_config.yml#L8) configuration for the amount of days the registration will be open. Before I think we had 10 days.